tests/unit_tests.py

```diff
@@ -477,51 +496,34 @@ def test_chinese_fulltext_extract(self):
         article.download(html)
         article.parse()
         text = mock_resource_with('chinese', 'txt')
-        assert article.text == text
-        assert fulltext(article.html, 'zh') == text
+        self.assertEqual(text, article.text)
+        self.assertEqual(text,  fulltext(article.html, 'zh'))
 
     @print_test
     def test_arabic_fulltext_extract(self):
-        url = 'http://arabic.cnn.com/2013/middle_east/8/3/syria.clashes/'\
+        url = 'http://arabic.cnn.com/2013/middle_east/8/3/syria.clashes/' \
               'index.html'
         article = Article(url=url)
         html = mock_resource_with('arabic_article', 'html')
         article.download(html)
         article.parse()
-        assert article.meta_lang == 'ar'
+        self.assertEqual('ar', article.meta_lang)
         text = mock_resource_with('arabic', 'txt')
-        assert article.text == text
-        assert fulltext(article.html, 'ar') == text
+        self.assertEqual(text, article.text)
+        self.assertEqual(text, fulltext(article.html, 'ar'))
 
     @print_test
     def test_spanish_fulltext_extract(self):
-        url = 'http://ultimahora.es/mallorca/noticia/noticias/local/fiscal'\
+        url = 'http://ultimahora.es/mallorca/noticia/noticias/local/fiscal' \
               'ia-anticorrupcion-estudia-recurre-imputacion-infanta.html'
         article = Article(url=url, language='es')
         html = mock_resource_with('spanish_article', 'html')
         article.download(html)
         article.parse()
         text = mock_resource_with('spanish', 'txt')
-        assert article.text == text
-        assert fulltext(article.html, 'es') == text
+        self.assertEqual(text, article.text)
+        self.assertEqual(text, fulltext(article.html, 'es'))
 
 
 if __name__ == '__main__':
-    # unittest.main()  # run all units and their cases
-
-    suite = unittest.TestSuite()
-
-    if len(sys.argv) > 1 and sys.argv[1] == 'fulltext':
-        suite.addTest(ExhaustiveFullTextCase())
```

Comment:
@amorgun @codelucas Removing this makes the tests runs "fulltext" every time... not sure if this is the best approach...
In any case, the readme should be updated to reflect this.